cmake/projects/brotli/hunter.cmake

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2016-2020, Rahul Sheth, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    brotli
+    VERSION
+    1.0.9-p0
+    URL
+    "https://github.com/drodin/brotli/archive/refs/heads/hunter-1.0.9.zip"
```

Comment:
@rbsheth please create a fork of https://github.com/google/brotli at https://github.com/cpp-pm/brotli for @drodin to access and use

Reply:
https://github.com/cpp-pm/brotli

Reply:
@rbsheth @NeroBurner I don't have access rights to this repo.

Reply:
Sorry, fixed @drodin

Reply:
@rbsheth Still can't push a new branch:
```
remote: error: GH006: Protected branch update failed for refs/heads/hunter-1.0.9.
remote: error: At least 1 approving review is required by reviewers with write access. You're not authorized to push to this branch. Visit https://docs.github.com/articles/about-protected-branches/ for more information.
To github.com:cpp-pm/brotli.git
 ! [remote rejected] hunter-1.0.9 -> hunter-1.0.9 (protected branch hook declined)
```

Reply:
Wow just saw this, sorry @drodin. You can't push to a `hunter*` branch directly after it has been created, so try pushing to another branch name and creating a PR for review.

Reply:
Hi @rbsheth. I want to revive this PR. There was never before a branch `hunter*` created in this repo. I'm sure that I was able to push to a new `hunter*` branch in other repos. I was able to push to brunch `pr.hunter-1.0.9`. What should I choose as a target for a PR if there is no `hunter*` branch? Can't rename `pr.hunter-1.0.9` branch to `hunter-1.0.9` as well...

Reply:
Hi @drodin, not sure what happened so I made a `hunter-1.0.9` branch for you https://github.com/cpp-pm/brotli/tree/hunter-1.0.9